keyboards/ergo42/keymaps/hdbx/keymap.c

```diff
@@ -11,20 +11,38 @@ extern keymap_config_t keymap_config;
 
 // レイヤー
 #define _QWERTY 0
-#define _LOWER 1
-#define _RAISE 2
-#define _GAME 3
+#define _GAME 1
+#define _LOWER 2
+#define _RAISE 3
 #define _ADJUST 4
 
 enum custom_keycodes {
-  QWERTY = SAFE_RANGE, // デフォルトレイヤー用
+  QWERTY = SAFE_RANGE, // QWERTYレイヤーへ
+  GAME,                // GAMEレイヤーへ
   MCR1,                // マクロ1
   MCR2,                // マクロ2
   MCR3,                // マクロ3
   DYNAMIC_MACRO_RANGE, // ダイナミックマクロ
   WN_SCLN,             // タップでJISの「:」  シフトでJISの「;」 (Windows)
 };
 
+// RGB Light Setting
```

Comment:
You can find a master list here, actually: https://github.com/qmk/qmk_firmware/blob/master/quantum/rgblight_list.h

Reply:
Thanks. I use master list.